src/views/StackView/StackViewTransitionConfigs.tsx

```diff
@@ -78,8 +78,14 @@ const NoAnimation = {
 function defaultTransitionConfig(
   transitionProps: TransitionProps,
   prevTransitionProps?: TransitionProps,
-  isModal?: boolean
+  isModal?: boolean,
+  isTransparent?: boolean
 ): TransitionConfig {
+  // If modal is not full-screen, user is responsible to implement animation
+  if (isModal && isTransparent) {
```

Comment:
Should we add an `Platform.OS === 'android'` check here since the animation works well on iOS?

Reply:
As discussed with @brentvatne on https://github.com/react-navigation/stack/issues/95#issuecomment-475732336 we decided to remove animation to unify both platforms...